website/addons/wiki/utils.py

```diff
@@ -35,6 +35,48 @@ def get_sharejs_uuid(node, wname):
     )) if private_uuid else None
 
 
+def delete_share_doc(node, wname):
```

Comment:
These were tied to wiki page objects, which didn't exist until they were saved. By moving the functionality off of the wiki page object, it enables migrations and deletions for wikis that have not yet been saved.